### PR TITLE
[16.0][MIG] hr_holidays_settings

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

File: hr_holidays_settings/i18n/hr_holidays_settings.pot [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* hr_holidays_settings
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
```
Comment (MiquelRForgeFlow) [inline hr_holidays_settings/i18n/hr_holidays_settings.pot:7]:
```suggestion
"Project-Id-Version: Odoo Server 16.0\n"
```

File: hr_holidays_settings/views/res_config_settings.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            </xpath>
+        </field>
+    </record>
+    <record id="hr_holidays_config_settings_action" model="ir.actions.act_window">
+        <field name="name">Configure HR Holidays</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">res.config.settings</field>
+        <field name="view_mode">form</field>
+        <field name="target">inline</field>
+        <field name="context">{'module' : 'hr_holidays'}</field>
```
Comment (MiquelRForgeFlow) [inline hr_holidays_settings/views/res_config_settings.xml:30]:
```suggestion
        <field name="context">{'module': 'hr_holidays'}</field>
```

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
@pedrobaeza  @Shide can you merge this PR?

**Comment by gjotten** [comment]
@JordiBForgeFlow @pedrobaeza I am now confronted with failing builds:

`AddonsConfigError: Addon hr_holidays_settings defined in several repos {'OCA/hr-holidays', 'OCA/hr'}`

https://github.com/OCA/hr/pull/1244 added the module to the `hr` repo; here we introduce it a second time. Looks like one place too many.

**Comment by pedrobaeza** [comment]
Deleting it in https://github.com/OCA/hr/pull/1391

**Pull request merged**